src/pages/help-centre/data/_ib-programme.ts

```diff
@@ -89,8 +123,10 @@ const ib_programme: TQuestionsData = {
             label: 'receive-commission',
             answer: [
                 {
-                    translation_text:
+                    eu_translation_text:
                         "_t_We'll pay your IB commission into your MT5 Synthetic account daily._t_",
+                    translation_text:
+                        "_t_We'll pay your IB commission into your MT5 (Derived account daily)._t_",
```

Comment:
Better to follow one approach 
I noticed in src/pages/help-centre/data/_deposits-and-withdrawals.ts we removing it at all
Is it possible?